src/ruleset.ts

```diff
@@ -29,6 +29,36 @@ export default {
             }
 
         },
+        "aep-131-response-schema": {
+            description: "Verifies that the response is an AEP resource.",
+            message: "The response body does not include an AEP resource.",
+            severity: DiagnosticSeverity.Error,
+            formats: [oas3],
+            given: "$.paths[*].get.responseBody.content[application/json].schema",
```

Comment:
we wiil overlook this check if the content type isn't application/json, is that ok? I guess we document that we should accept application/json, but we don't officially require it in the bullets below: https://aep.dev/131#requests.